user_hello_mobile.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Hello Swarm on mobile
+about: Hello Swarm on mobile
+title: Hello Swarm on mobile
+---
+
+## Rationale ##
+
+## User-Story ##
+
+I want to be able to run Swarm/PSS on mobile and get a basic end-to-end hello world going. This doesn't include connectivity strategies etc, and is expected to be naive.
```

Comment:
Apparently every user story needs to clarify who is the customer. e.g. "As a node operator I want my node to do XYZ" or "As a dapp developer I want to be able to XYZ" so please rephrase :)

Reply:
fixed